tests/test_model_setups/test_hf.py

```diff
@@ -0,0 +1,31 @@
+"""Unit tests for the `model_setups.hf` module."""
+
+import pytest
+import torch
+from scandeval.model_setups.hf import HFModelSetup
+
+
+@pytest.mark.parametrize(
+    argnames=["model_id", "expected"],
+    argvalues=[
+        ("microsoft/mdeberta-v3-base", None),
+        ("jonfd/electra-small-nordic", None),
+        ("mistralai/Mistral-7B-v0.1", torch.bfloat16),
```

Comment:
Seems like quite large models to download for tests. I would probably:

```suggestion
        ("microsoft/deberta-v3-xsmall", None),
        ("jonfd/electra-small-nordic", None),
        ("mistralai/Mistral-7B-v0.1", torch.bfloat16),
```
Note it is not multilingual.

Is there a smaller alternative for the mistral (I couldn't find one). Probably a good point going forward though: To always make and release very small models for tests.

Reply:
It's only downloading the model configs, not the models, so it should be alright :)

Reply:
Ah perfect, then just keep it as is.